data/test-cases/plugins/gitlab/subscriptions-and-notifications/Issue_link_tooltip.md

```diff
@@ -0,0 +1,71 @@
+---
+# (Required) Ensure all values are filled up
+name: "To verify the Tooltip upon hovering cursor over the link for issues."
+status: Active
+priority: Normal
+folder: Subscriptions-and-notifications
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Create an issue in the desired project and copy its link on the Gitlab.
+2. Navigate to MM and paste the link of the created issue in any desired channel or DM/GM on MM.
+3. Hover the cursor over the link of the created issue on MM.
+
+**Step 2**
+
+1. After step 1, close the created issue on Gitlab.
+2. Navigate to MM and refresh the page.
+3. Hover the cursor the link of the closed issue on MM.
+
+**Step 3**
+
+1. Delete any desired issue on Gitlab whose link exist in any desired channel or DM/GM on MM.
+2. Navigate to MM and refresh the page.
+3. Hover the cursor over the link of the deleted issue on MM.
+
+**Step 4**
+
+1. Crete a link for a non existing issue on Gitlab and paste it in any desired channel or DM/GM on MM.
```

Comment:
Re-phrase this, non-existing link can not be created. This can be written as paste the link which does not exist on Github. Fix everywhere.

Reply:
Here i meant to say that, `create a link for a non-existing issue on Gitlab` not a `non-existing link for gitlab issue` please consider this and let me know if i have to update this.